vendor/justwatchcom/elasticsearch_exporter/examples/grafana/dashboard.json

```diff
@@ -1167,6 +1167,6 @@
     ]
   },
   "timezone": "utc",
-  "title": "ElasticSearch Example",
+  "title": "Elasticsearch",
```

Comment:
Could we use configuration of the aggregation to rewrite the name on the fly, instead of editing vendored stuff directly ?

Reply:
Good point, I didn't consider that... I'll make those changes.

Reply:
I now implemented this, thanks for the suggestion!